docker/prometheus.yml

```
@@ -1,6 +1,6 @@
-global:
-  scrape_interval: 15s
-scrape_configs:
-  - job_name: 'otel-collector'
-    static_configs:
-      - targets: ['localhost:8888']
+global:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)